resilience4j-retrofit/README.adoc

```diff
@@ -0,0 +1,39 @@
+= Retrofit Circuitbreaker
```

Comment:
please rename tittle and add copyright info to documentation and all your code files.